setup.py

```diff
@@ -82,6 +83,7 @@ def recursive_data_files(directory, install_directory):
     "pytest-mock",
     "pytest-subprocess",
     "ruff",
+    "tox>=4.0",
```

Comment:
Is this supposed to align with the version in the tox.yaml workflow?
`python -m pip install 'tox>=4.0.11<5.0' tox-gh`

Reply:
Yes/no? I wanted to be more explicit in the yaml file that we don't want to do a major release upgrade, but here I was trying to fit with our current theme of the least restrictive possible requirements - at the time I wrote this, 4.0 were all still considered prerelease so `pip install tox` would install 3.x. It looks like as of 4.1 that's no longer the case, but also I know for a fact that our config doesn't work with 3.x.

Reply:
Long term I plan on having Renovate update the versions in the yaml file as well though :-)